class.jetpack.php

```diff
@@ -486,6 +486,8 @@ private function __construct() {
 			Jetpack_Heartbeat::init();
 		}
 
+		add_action( 'rest_authentication_errors', array( $this, 'wp_json_authenticate' ) );
```

Comment:
Good to know, thanks for sharing. This will not likely be a problem for us, since requests will be run through a proxy endpoint on wpcom, allowing us to monitor the requested endpoint, even when the authentication fails.